admin/wait-sqldb.sh

```diff
@@ -5,6 +5,6 @@ if [[ "${WAIT_FOR_SQL_SERVER:=false}" != "false" ]]; then
 
   until [[ "$(curl -sS $SQL_SERVER:$SQL_PORT 2>/dev/stdout)" =~ "(52)" ]]; do
```

Comment:
I don't think anything is actually using this script.

Reply:
this could well have been something that was added by DW when we were exploring the ruby in docker thing a while back